built-in-references/Kubernetes/gatekeeper-opa-sync.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: config.gatekeeper.sh/v1alpha1
+kind: Config
+metadata:
+  name: config
+  namespace: "gatekeeper-system"
```

Comment:
this namespace needs to be configurable or this resource will be ignored. it should be the same namespace as where the gatekeeper components live. so let's remove this namespace from this file, then in the doc, add something like:

```
kubectl apply -f sync.yaml -n <GATEKEEPER NAMESPACE>
```

cc @sozercan

Reply:
If you are planning to add this sync.yaml as part of the chart, then ^ doesn't matter as much since we can assume gatekeeper is in `gatekeeper-system` if they use the gatekeeper.yaml from the gatekeeper repo.